src/Tribe/Ticket_Object.php

```diff
@@ -1089,16 +1089,13 @@ public function get_event_id() {
 		 * @return bool Whether the ticket has meta enabled.
 		 */
 		public function has_meta_enabled() {
-			$has_meta = get_post_meta( $this->ID, '_tribe_tickets_meta_enabled', true );
-			$has_meta = ! empty( $has_meta ) && tribe_is_truthy( $has_meta );
-
 			/**
 			 * Filters if the ticket has meta or not.
 			 *
-			 * @param bool $has_meta Wether if it has meta.
+			 * @param bool $has_meta Whether the ticket has meta enabled.
 			 * @param int  $ticket_id The ticket ID.
 			 */
-			return (boolean) apply_filters( 'tribe_tickets_has_meta_enabled', $has_meta, $this->ID );
+			return (boolean) apply_filters( 'tribe_tickets_has_meta_enabled', false, $this->ID );
```

Comment:
:no_entry_sign: Short form type keywords must be used. Found: (boolean)

<sub>hash: ae1af48a16db019f861c1f3099f395a2</sub>

Reply:
```suggestion
			return (bool) apply_filters( 'tribe_tickets_has_meta_enabled', false, $this->ID );
```